anyway/parsers/location_extraction.py

```diff
@@ -128,7 +126,7 @@ def get_bounding_box(latitude, longitude, distance_in_km):
         .filter(AccidentMarkerView.geom.intersects(polygon_str))
         .filter(AccidentMarkerView.accident_year >= cutoff_year)
         .filter(AccidentMarkerView.provider_code != BE_CONST.RSA_PROVIDER_CODE)
-        .filter(not_(AccidentMarkerView.road_segment_name == None))
+        .filter(not_(AccidentMarkerView.road_segment_name is None))
```

Comment:
Notice it's not a pythonic operation but SqlAlchemy, from reading this message on Stackoverflow(below) i don't believe it will work

https://stackoverflow.com/a/5632224/13771969